api.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ *	Report API endpoint.
+ */
+
+namespace Nottingham\AdvancedReports;
+
+header( 'Content-Type: application/json' );
+
+
+// Verify the report exists and is accessible through the API.
+$reportID = $_GET['report_id'];
+$listReports = $module->getReportList();
+if ( ! isset( $listReports[$reportID] ) || ! isset( $listReports[$reportID]['as_api'] ) ||
+     ! $listReports[$reportID]['as_api'] )
+{
+	http_response_code( 400 );
+	echo 'false';
```

Comment:
sorry - showing my ignorance here I know, but is the echo 'false' just debug code, and does it output anything anywhere on live systems where perhaps we don't want it, or it may build up over time?

Reply:
This page is the API endpoint for Advanced Reports. If `false` is returned (along with the 400 status code), it indicates the API request was invalid. This could be because:
* The specified report ID does not exist.
* The report does not support API access, or API access is not enabled.
* The API key is incorrect.